src/com/company/Verification.java

```diff
@@ -0,0 +1,57 @@
+package com.company;
+
+import java.util.*;
+
+public class Verification {
+
+    public static long verifyPhoneNumber(Scanner sc){
+        long number;
+        do {
+            while(!sc.hasNext()){
+                System.out.println("Enter a valid phone number!");
+                sc.next();
+            }
+           while(!sc.hasNextLong()){
+               System.out.println("Enter a valid phone number!");
+               sc.next();
+           }
+           number = sc.nextLong();
+       }while(number<=0);
+       sc.nextLine();
```

Comment:
Seems like here, we are just validating if we have any input and if that input is a number. Here, we also need to verify that the number entered is 10 digit long or not.